buzsaki_lab_to_nwb/original-code/convert_yuta.py

```diff
@@ -221,29 +219,109 @@ def yuta2nwb(session_path='/Users/bendichter/Desktop/Buzsaki/SenzaiBuzsaki2017/Y
     ns.add_position_data(nwbfile, session_path)
 
     shank_channels = ns.get_shank_channels(session_path)[:8]
+    nshanks = len(shank_channels)
     all_shank_channels = np.concatenate(shank_channels)
 
     print('setting up electrodes...', end='', flush=True)
     hilus_csv_path = os.path.join(fpath_base, 'early_session_hilus_chans.csv')
     lfp_channel = get_reference_elec(subject_xls, hilus_csv_path, session_start_time, session_id, b=b)
-    print(lfp_channel)
+    
+    
     custom_column = [{'name': 'theta_reference',
                       'description': 'this electrode was used to calculate LFP canonical bands',
                       'data': all_shank_channels == lfp_channel}]
     ns.write_electrode_table(nwbfile, session_path, custom_columns=custom_column, max_shanks=max_shanks)
+    
+    print('reading raw electrode data...', end='', flush = True)
+    if stub:
+        # example recording extractor for fast testing
+        num_channels = 4
+        num_frames = 10000
+        X = np.random.normal(0, 1, (num_channels, num_frames))
+        geom = np.random.normal(0, 1, (num_channels, 2))
+        X = (X * 100).astype(int)
+        sre = se.NumpyRecordingExtractor(timeseries=X, sampling_frequency=20000, geom=geom)
+    else:
+        nre = se.NeuroscopeRecordingExtractor('{}/{}.dat'.format(session_path,session_id))
+        sre = se.SubRecordingExtractor(nre,channel_ids=all_shank_channels)
+        
+    print('writing raw electrode data...', end='', flush=True)
+    se.NwbRecordingExtractor.add_electrical_series(sre,nwbfile)
```

Comment:
This was the intermediate refactor that uses the auxiliary functions in SpikeInterface to add only the raw electrical series, since the rest of this code also handled the device, electrode groups, etc. The main version on the other branch uses the new full version of `se.NwbRecordingExtractor.write_recording(...)` with metadata to handle all that.